### DISCUSSION TITLE
Using a single model for modeling multiple time series

### DISCUSSION THREAD

**Discussion by Terabyte17** [ourownstory/neural_prophet#1194] [Q&A - forecasting best practices] [answered]
Say, I have multiple time series of sales of different products in a store, is it possible to model them using a single NeuralProphet model? Something like generating embeddings for each product and then passing them as lagged regressors to the model? If yes, then how do I fit the model on those multiple time series?

**Comment by Kevin-Chen0** [comment, marked as answer]
Yes, we have something called **global modeling**. It is basically a single model forecasting a dataset that is comprised of multiple time series by _sharing weights_.

You can see tutorials on how to use it [here](https://github.com/ourownstory/neural_prophet/blob/main/tutorials/feature-use/global_modeling.ipynb) and [here](https://github.com/ourownstory/neural_prophet/blob/main/tutorials/feature-use/global_local_modeling.ipynb).

**Reply by M-Adrian-N** [reply]
Many thanks Kevin!
